### ISSUE TITLE
Add ability to edit Custom fields

### ISSUE THREAD

**Issue by vabue** [snipe/snipe-it#1684]
In 2.10 when you made custom asset once - that's all, you can't edit it. 

But changing, at least, the name of the field would be very useful.

**Comment by zmorrow** [comment]
I'd like to request this as well, I made a custom field required and would like to go back and change that if possible.

**Comment by sphere720** [comment]
I can't seem to even find where I can enter data for the custom fields I created???

**Comment by sphere720** [comment]
Nevermind. They showed up when I switched the asset model to something else and then switched it back. I agree with vabue and zmorrow about being able to change these settings after creating them. I've been leary about making any of them required for this reason. Other than that, and the importing feature we're all waiting on; this isht is freaking awesome!

**Comment by SylentBobNJ** [comment]
+1 this

**Comment by snipe** [comment]
This is already planned for development

**Comment by sphere720** [comment]
You rock!

**Comment by tiagom62** [comment]
Has there been any progress on this?
 
If not, I would like to try and implement the feature. As I needed this today.

**Comment by snipe** [comment]
No progress so far.

**Comment by JordanC12** [comment]
@snipe Do we think this feature will be available in one of the upcoming releases? As of now, every time I redeploy an asset back to a user I need to delete the asset and re-import the asset since I cant find a way to edit the (custom) fields on a checked in asset

**Comment by grandmamoses** [comment]
This is very important for us too. I have a custom field list which I need to add items to the list, but can't.

Thanks!

**Comment by crazycga** [comment]
I'd like to throw my own hat in the ring on this, too.

**Comment by WienTourismus** [comment]
Please put this topic on top of the development issues as it it critical for us to add items to the listbox of a custom field

**Comment by jenrobff** [comment]
+1 for this , I feel like this is a much needed feature. Please and thank you :).

**Comment by reison007** [comment]
Hi,

Yeah, I agree. ; )

2017-01-19 22:11 GMT-02:00 jenrobff <notifications@github.com>:

**Comment by yguvvala** [comment]
was this developed and is there an update on this ?

**Comment by snipe** [comment]
<img width="741" alt="screen shot 2017-02-02 at 4 10 02 pm" src="https://cloud.githubusercontent.com/assets/197404/22574164/17afa06e-e962-11e6-84b4-94d79c3987a1.png">

It's on develop, and will be merged into the next release.

**Comment by sphere720** [comment]
Because you're cool like that.

Donald Counts

**Comment by seanad** [comment]
Any word on when this will make it to live? It was mentioned it would be merged into the next release, but I installed the latest release and don't appear to have this functionallity.

**Comment by snipe** [comment]
It will be in v4 when that comes out

**Comment by petestanley** [comment]
Could really do with this ability

**Comment by snipe** [comment]
@petestanley It will be in v4 when that comes out

**Comment by sickyazone** [comment]
Can't wait for this feature to release. 👍 
Any ETA on v4? :)
